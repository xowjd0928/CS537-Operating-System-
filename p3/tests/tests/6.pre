LANG=C ls -1 ../solution | tr -d '*' > tests/6.out
