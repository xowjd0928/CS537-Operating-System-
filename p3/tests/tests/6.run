../solution/wsh <(echo cd ../solution; echo ls) | tr -d '*'
