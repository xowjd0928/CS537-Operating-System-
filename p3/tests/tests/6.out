Makefile
wsh
wsh-dbg
wsh.c
wsh.h
